python/src/com/jetbrains/python/psi/types/PyModuleType.java

```diff
@@ -137,20 +161,23 @@ private static PsiElement resolveByOverridingMembersProviders(@NotNull PyFile mo
   }
 
   @Nullable
-  private List<? extends RatedResolveResult> resolveImplicitPackageMember(@NotNull String name,
-                                                                          @NotNull List<PyImportElement> importElements) {
-    final VirtualFile moduleFile = myModule.getVirtualFile();
+  private static List<? extends RatedResolveResult> resolveImplicitPackageMember(@NotNull PsiFileSystemItem moduleOrPackage,
```

Comment:
I agree, `resolveImplicitPackageMember` and `collectImportedSubmodules` do look similar. Not sure how to refactor them to remove this duplication though. Feel free to skip it if you're not sure too (a TODO item might be OK for now).